source/mobile/mobile-troubleshoot.rst

```diff
@@ -68,6 +68,20 @@ A server running Linux has this file located in ``/etc/mime.types``. This might
 
 Some distributions also ship without ``mailcap`` which can result in missing or incorrectly configured mime types.
 
+Messages with emojis aren't being sent from the Mobile App
+----------------------------------------------------------
+
+This can occur if the server running Mattermost is configured with an incorrect character set. To resolve this issue, in the ``config.json`` file under ``SqlSettings``, ensure that the ``DataSource`` key is configured correctly, then restart the Mattermost server. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#data-source>`__ documentation for details.
+
+For example:
+
+.. code-block:: none
+
+  "SqlSettings": {
+      "DataSource": "<user:pass>@<servername>/mattermost?charset=utf8mb4,utf8",
```

Comment:
@agnivade - Can you provide updated examples for a) using collation, b) letting the library use the default collation, c) the default collation for mySQL 8.0, please? While I won't include all 3 examples here, I will include them in the configuration settings documentation.

Reply:
1. Using collation:
`?charset=utf8mb4,utf8&collation=utf8mb4_general_ci`
2. Letting the library use the default collation, which is `utf8mb4_general_ci`
`?charset=utf8mb4,utf8`
3. Using the new collation for MySQL 8.0
`?charset=utf8mb4,utf8&collation= utf8mb4_0900_ai_ci`